src/ResourceManager/AzureBatch/Commands.Batch/BatchAccounts/NewBatchAccountCommand.cs

```diff
@@ -40,13 +42,30 @@ public class NewBatchAccountCommand : BatchCmdletBase
         [Parameter(Position = 3, Mandatory = false, ValueFromPipelineByPropertyName = true)]
         public string AutoStorageAccountId { get; set; }
 
+        [Parameter(Position = 4, Mandatory = false, ValueFromPipelineByPropertyName = true)]
```

Comment:
If we don't set position, will it reduce the breaking change for future?

Reply:
I am not sure -- I followed the existing pattern in this cmdlet. @cormacpayne @markcowl can you comment on this?

Reply:
Think of this like unnamed parameters in a method signature - any positional parameter beyiond 3 tends to be not useful.  We suggest that you not have positional parameter with position > 3

Reply:
Got it. I will fix this